rotkehlchen/tests/unit/test_tokens.py

```diff
@@ -211,9 +212,9 @@ def _do_spawn(database):
     while True:
         gevent.spawn(_do_read, database)
         with database.user_write() as write_cursor:
-            database.set_setting(write_cursor, 'last_write_ts', 15)
+            database.set_cache(write_cursor, DBCache(last_write_ts=Timestamp(15)))
```

Comment:
same comment. With this approach I believe you are overwritting all the settings. Check it